crates/nostr-relay-builder/src/builder.rs

```diff
@@ -215,6 +217,7 @@ impl Default for RelayBuilder {
             #[cfg(feature = "tor")]
             tor: None,
             max_connections: None,
+            max_subid_length: None,
```

Comment:
Yeah, I think we can set a default limit here. I guess we can also remove the option

Reply:
250?